cmd/govim/testdata/scenario_default/config_set_env_goflags_build_tags.txt

```diff
@@ -0,0 +1,50 @@
+# Test that calling govim#config#Set with a value for GoplsEnv of GOFLAGS
+# does the right thing.
+
+[golang.org/issues/29202] skip
+
+# Before setting GOFLAGS with -tags=other
+vim ex 'e main.go'
+vimexprwait pre.golden getqflist()
+
+# After setting GOFLAGS with -tags=other
+vim ex 'call govim#config#Set(\"GoplsEnv\", {\"GOFLAGS\": \"-tags=other\"})'
+vimexprwait post.golden getqflist()
+
+# Assert that we have received no error (Type: 1) or warning (Type: 2) log messages
+# Disabled pending resolution to https://github.com/golang/go/issues/34103
+# errlogmatch -start -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:(1|2), Message:".*'
+
+-- go.mod --
+module mod.com
+
```

Comment:
We still need to add `go 1.12` to avoid #36243 right?

Reply:
Great spot. Thank you.